portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase

```diff
@@ -455,13 +455,93 @@ definition {
 		}
 	}
 
-	@description = "This test anonymizes a staged and a live entry from an applciation."
-	@ignore = "Test Stub"
+	@description = "This test anonymizes a staged and a live entry from an applciation, test with WC entries."
 	@priority = "3"
-	test CanAnonymizeASingleStagedAndLiveEntryFromAnApplication {
+	test CanAnonymizeASingleStagedAndLiveEntryFromWebContent {
+		task ("Given log in as a new user, create a site enable local staging") {
+			User.logoutPG();
+
+			User.firstLoginUI(
+				password = "test",
+				userEmailAddress = "userea@liferay.com");
+
+			JSONGroup.addGroup(groupName = "Site Name");
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Staging Test Page");
+
+			JSONUser.addUserToSite(
+				groupName = "Site Name",
+				userEmailAddress = "userea@liferay.com");
+
+			JSONStaging.enableLocalStaging(groupName = "Site Name");
+		}
+
+		task ("Given add the staged entry with web content and publish to live this entry") {
+			for (var count : list "1,2,3") {
+				JSONWebcontent.addWebContent(
+					content = "WC Content ${count}",
+					groupName = "Site Name (Staging)",
+					site = "false",
+					title = "GDPR Anonymize Article ${count}",
+					userEmailAddress = "userea@liferay.com",
+					userPassword = "test");
+			}
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name-staging");
 
-		// TODO LRQA-50756 Anonymize a single staged and live entry from an Application
+			Staging.publishToLiveNowViaPortletCP();
+		}
+
+		task ("When sign in as Test Test and deactivate the user usersn.") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
 
+			User.openUsersAdmin();
+
+			User.deletePersonalDataCP(
+				userFirstName = "userfn",
+				userLastName = "userln");
+		}
+
+		task ("And select anonymize staged 'GDPR Anonymize Article 1' entry and lived 'GDPR Anonymize Article 2' entry") {
```

Comment:
If it's hard to combine the When, using And to nested it like following.
task ("When balbalbalbla")
    task ("And balbalbalbla")